website/views.py

```
@@ -107,3 +107,16 @@ def delete_record(request, pk):
     except Exception as _:
         messages.error(request, "An error occurred while deleting the record")
         return redirect('home')
+
+def add_record(request):
+    form = AddRecordForm(request.POST or None)
+    if request.user.is_authenticated:
+        if request.method == "POST":
+            if form.is_valid():
```

Ruff (0.8.0)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)